src/Bank/BankLogic/AccountSettings/Owner.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace BankingSystem.BankLogic
+{
+    public class Owner : IOwnerStrategy
```

Comment:
Rename the interface to get rid of "Strategy" part

Reply:
I hope what "IAccountOrigin" as name will be fine